docs/CODING_GUIDELINES_ANGULAR.md

```diff
@@ -157,16 +157,16 @@ ClrFormsModule
 
 ### Public API
 
-* Custom texts should almost always be received through content projection, **not** inputs. This will allow for icons, images, links, etc.
+* Custom texts should almost always be received through content projection, **not** inputs. This convention allows icons, images, links, etc.
```

Comment:
How about for second sentence something like:

This convention facilitates usage of icons, images, links, etc as custom content.